lib/kino/table.ex

```diff
@@ -92,6 +94,8 @@ defmodule Kino.Table do
       This works the same as `Kino.JS.new/3`, except the function
       receives the state as an argument
 
+    * `:types` - a map of type overrides for the columns.
+      The keys are the column names and the values are the types to be used in place of inferred ones.
```

Comment:
```suggestion
    * `:types` - a map of display type overrides for the columns. The keys
      are the column names and each value must be one of `t:type/0`. By
      default the types are inferred from the data values
```

The user needs to know what types are available and I would link to `t:type/0` to keep it in a single place.